spec/javascripts/live_search_spec.js

```diff
@@ -170,6 +174,44 @@ describe('liveSearch', function () {
     expect(liveSearch.cache('some-slug')).toBe('something in the cache')
   })
 
+  describe('should update the URL correctly', function () {
+    beforeEach(function () {
+      // clear these options to simplify the URLs
+      $form.find('input[name=field]').prop('checked', false)
+      $form.find('input[name=published_at]').val('')
+    })
+
+    it('when search options change', function () {
```

Comment:
This reads a bit strange as "it when search options change" I can see it's a continuation of "should update the URL correctly" but would suggest doing something like:

describe('updating the URL correctly')
it('should update when the search options change')
it('should update when there is a cached response')

Reply:
Fair enough. Have tweaked, now looks like this:

![Screenshot 2021-08-09 at 08 44 00](https://user-images.githubusercontent.com/861310/128673910-49be917c-75cd-4880-a93f-ffa39b142574.png)